astropy/io/fits/diff.py

```diff
@@ -417,17 +417,13 @@ def _report(self):
         if self.ignore_hdu_patterns:
             ignore_hdu_patterns = " ".join(sorted(self.ignore_hdu_patterns))
             self._writeln(
-                " HDU(s) not to be compared:\n{}".format(
-                    wrapper.fill(ignore_hdu_patterns)
-                )
+                f" HDU(s) not to be compared:\n{wrapper.fill(ignore_hdu_patterns)}"
```

Comment:
`wrapper.fill()` returns `str` so I think this would be clearer as
```
" HDU(s) not to be compared:\n" + wrapper.fill(ignore_hdu_patterns)
```
And same for next two.

But no big deal.

Reply:
Addressed here and duplicated for surrounding messages.